lib/components/avatar.css

```diff
@@ -0,0 +1,63 @@
+.av {
+    border-radius: 100%;
+    height: 4.5rem;
+    width: 4.5rem;
+}
+
+.av-sm,
+.av-md, 
+.av-lg, 
+.av-xl {
+    border-radius: 100%;
+}
```

Comment:
you can add .av in the common class code and remove the `border-radius` from `.av class` so the code is not redundant

Reply:
Done.
Thanks for pointing this.